scripts/population/load-regions

```diff
@@ -33,13 +33,30 @@ if [[ ! -e ${DATA_PATH}/$COUNTRY_CODE ]]; then
   exit 1
 fi
 
+# Simplification parameter
+# From mapshaper wiki https://github.com/mbloch/mapshaper/wiki/Command-Reference#-simplify
+#
+# interval= Specify simplification amount in units of distance.
+#           Uses meters when simplifying unprojected datasets in 3D space (see planar option below),
+#           otherwise uses the same units as the source data.
+INTERVAL_DISTANCE=${INTERVAL_DISTANCE-1000}
```

Comment:
Could you please expand the comment above explaining that in our case since our GeoJSON are not projected (ie. use WGS84 coordinates) then this means the resolution for the simplified shape will be 1km? (Or at least that's what I understand from the documentation :grin:)

Reply:
I wasn't sure about what the parameter was measuring in this case. I compared the mapshaper output of the Kenya map or 100 and 1000 (it is the island on the south east)

<img width="739" alt="Screenshot 2024-05-30 at 10 42 48 PM" src="https://github.com/instedd/planwise/assets/79440/c78cd52e-9030-47e5-a8ca-dce0af270e2c">

<img width="618" alt="Screenshot 2024-05-30 at 10 46 03 PM" src="https://github.com/instedd/planwise/assets/79440/97c3696a-ccb6-47c8-b1e5-1b158a937fbc">

Reply:
Hmm... I'm thinking this can potentially lead to legitimate providers being excluded. Maybe we should play a bit with the thresholds and the problematic countries and check the resulting sizes. We may get away with a smaller threshold. @manumoreira what was the country that triggered the original issue?

Reply:
In fact, a default of 100 may work fine. I'm testing locally with some datasets I have and Philipines shrinks from 58 Mb to 2.5 Mb using 100 as the threshold. That should be acceptable I think, since Kenya's contour is 4.9 Mb unprocessed and we didn't have problems with that country so far. NB. sizes are of the admin level 0 `.geojson` files.

Reply:
The country was Mozambique

Reply:
Comparison between the original and the 100 meters simplification

<img width="841" alt="Screenshot 2024-05-31 at 10 14 44 AM" src="https://github.com/instedd/planwise/assets/79440/3f0dcc50-6d4c-471a-84a0-041a73f77cd9">

Reply:
I had to zoom more because the difference wasn't visible from https://mapshaper.org/.

Reply:
Comparison for Mozambique at the north betwee original resolution and 100m maps

<img width="789" alt="Screenshot 2024-06-03 at 4 22 14 PM" src="https://github.com/instedd/planwise/assets/79440/21651107-80d4-4ad6-9f2a-0a6f3d578df0">

Testing files 
[moz-geojson.tgz](https://github.com/user-attachments/files/15539419/moz-geojson.tgz). I used https://mapshaper.org/ to load them in separate layers.